tests/braille-specs/zhcn-cbs.yaml

```diff
@@ -0,0 +1,74 @@
+# encoding: utf-8 
+#
+# Chinese Common Braille Scheme
+#
+# see Ministry of Education of the People's Republic of China's
+# website <http://www.moe.gov.cn/jyb_sjzl/ziliao/A19/201807/W020180725666187054299.pdf>
+#
+# Copyright © 2022 by Cary-rowen <manchen_0528@outlook.com>
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+#
+# ----------------------------------------------------------------------------------------------
+
+display: unicode-without-blank.dis
+table:
+  language: cmn-Hans
+  region: cmn-CN
+  system: cmn-traditional
+  variant: chinese-common-braille-scheme
+  __assert-match: zhcn-cbs.ctb
+tests:
+  - [苹果,⠏⠡⠛⠕⠄]
+  - [电脑,⠙⠩⠝⠖⠄]
+  - [橘子,⠛⠬⠂⠵]
+  - [眼睛,⠩⠄⠛⠡⠁]
+  - [杯弓蛇影,⠃⠮⠁⠛⠲⠁⠱⠢⠂⠡⠄]
+  - [老马识途,⠇⠖⠄⠍⠔⠄⠱⠂⠞⠥]
+  - [相敬如宾,⠓⠭⠁⠛⠡⠚⠥⠃⠣⠁]
+  - [飞黄腾达,⠋⠮⠓⠶⠞⠼⠙⠔⠂]
+  - [如鱼得水,⠚⠥⠬⠂⠙⠢⠂⠱⠺⠄]
+  - [今天的天气不错,⠛⠣⠁⠞⠩⠁⠙ ⠞⠩⠁⠅⠊⠆⠃⠥⠉⠕⠆]
+  - [做人要有梦想,⠵⠕⠚⠴⠜⠳⠍⠼⠆⠓⠭⠄]
+  - [朝月亮出发,⠟⠖⠾⠇⠭⠟⠥⠁⠋⠔]
+  - [最深最重的爱，须与时日共成长。,⠵⠺⠱⠴⠁⠵⠺⠌⠲⠙ ⠪⠐ ⠓⠬⠁⠬⠄⠱⠂⠚⠆⠛⠲⠟⠼⠌⠦⠄⠐⠆]
+  - [这是我的国土,⠌⠢⠱⠕⠙ ⠛⠕⠂⠞⠥⠄]
+  - [hello,⠓⠑⠇⠇⠕]
+  - [Hello World,⠠⠓⠑⠇⠇⠕ ⠠⠺⠕⠗⠇⠙]
+  - [战略合作协议,⠌⠧⠇⠾⠓⠢⠵⠕⠓⠑⠂⠊⠆]
+  - [123,⠼⠁⠃⠉]
+  - [中国是发展中国家,⠌⠲⠁⠛⠕⠂⠱⠋⠔⠌⠧⠄⠌⠲⠁⠛⠕⠂⠛⠫⠁]
+  - [尊老爱幼是中华民族的传统美德,⠵⠒⠁⠇⠖⠄⠪⠳⠆⠱⠌⠲⠁⠓⠿⠍⠣⠵⠥⠂⠙ ⠟⠻⠞⠲⠄⠍⠮⠄⠙⠢⠂]
+
+# The differences with the current braille system is illustrated with the following examples:
+
+  # some braille cells can be omitted without causing ambiguity:
+  - [他们,⠞⠍⠴] # requires only 3 braille cells instead of 4
+  - [它们,⠈⠞⠍⠴] # requires only 4 braille cells instead of 6
+  - [她们,⠞⠁⠍⠴] # requires only 4 braille cells instead of 5
+  - [你好,⠝⠓⠖⠄] # requires only 4 braille cells instead of 6
+
+  # some tones can be omitted:
+  - [海浪,⠓⠪⠄⠇⠦] # requires only 5 braille cells instead of 6
+  - [句子,⠛⠬⠵] #  requires only 3 braille cells instead of 5
+  - [巨资,⠛⠬⠵⠁] #  requires only 4 braille cells instead of 5
+  - [好朋友,⠓⠖⠄⠏⠼⠳] # requires only 6 braille cells instead of 8
+  - [好客,⠓⠖⠆⠅⠢] # requires only 5 braille cells instead of 6
+  - [备课,⠃⠮⠅⠢] # requires only 4 braille cells instead of 6
```

Comment:
Thanks. This helps a little bit to understand better the differences. It's not much, but it's something. I still don't quite get how it works. Probably that is because of my lack of knowledge of Chinese braille in general. Maybe you could explain "more perfect marking rules" a bit better for the layman like me. What exactly is marked? I understand that tones are a kind of marking that can be omitted when it causes no ambiguity. But what are other markings?

As you can see I've reformatted the YAML file a bit to make it easier to compare the tables.

Reply:
I missed your answer to this question in the [comment you made on August 11](https://github.com/liblouis/liblouis/pull/1233#issuecomment-1212178910). Just use this explanation in the table and test, then it's fine!